### PR TITLE
Use default config if exists

### PR DESCRIPTION
**Pull request by alexkvak**

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please do not open duplicate PR to force passes.

BTW, rules are sames, new tests are needed.

Duplicate of #95, closing.

**Comment by rande** [comment]
@alexkvak do you plan to add Tests ? if so you can take over the issue in this PR (we can reopen the PR)

**Comment by alexkvak** [comment]
yes, I write tests now

**Comment by rande** [comment]
@alexkvak great, thanks and waiting for your next commit. Please squash your commits

**Comment by alexkvak** [comment]
is it ok?

**Comment by rande** [comment]
@alexkvak CS tests are failing, see http://91.121.3.21/job/sonata-ci-bundle-formatter/9/console

**Comment by alexkvak** [comment]
OMG, trailing comma, seriously?

**Comment by rande** [comment]
@alexkvak we try to keep a clean code, that it.

**Comment by Soullivaneuh** [comment]
> OMG, trailing comma, seriously?

Yes, seriously. It's under Symfony convention.

BTW, it's pretty much easier to have comma on multi-lines array for clearer git diff.

**Comment by alexkvak** [comment]
Do you provide some tool that can fix CS according to the extra_empty_lines rule?

**Comment by rande** [comment]
yes, ``make cs`` .You will need to install php-cs-fixer in your path

**Comment by alexkvak** [comment]
> BTW, it's pretty much easier to have comma on multi-lines array for clearer git diff.
agree

**Comment by alexkvak** [comment]
> yes, make cs .You will need to install php-cs-fixer in your path
thanks, done

**Comment by alexkvak** [comment]
@rande everything passed

File: Form/Type/FormatterType.php [hunk 1]
```diff
@@ -129,9 +129,14 @@ public function buildView(FormView $view, FormInterface $form, array $options)
         }
         $view->vars['format_field_options'] = $options['format_field_options'];
 
-        $ckeditorConfiguration = array(
-            'toolbar'       => array_values($options['ckeditor_toolbar_icons']),
-        );
+        $defaultConfig = $this->configManager->getDefaultConfig();
+        if ($this->configManager->hasConfig($defaultConfig)) {
```
Comment (rande) [inline Form/Type/FormatterType.php]: I think you missed a case where the default configuration is set, and the ``ckeditor_toolbar_icons`` option is also set ?

Comment (alexkvak) [inline Form/Type/FormatterType.php]: yeah, you are right, it's not so pretty. I've just copied this code from old pull request..
Fixed.

**Comment by rande** [comment]
Not sure array_replace_recursive behave properly, can you add a test for this ? Also you need to squash your commits into one.

Thanks

**Comment by alexkvak** [comment]
> Not sure array_replace_recursive behave properly, can you add a test for this ? Also you need to squash your commits into one.

But it works properly

**Comment by rande** [comment]
@alexkvak thanks for the PR. next time, you can leave a comment to say the PR is ready. We don't receive any notification when you push a new change.

**Pull request merged**